Cargo.toml

```diff
@@ -253,3 +255,9 @@ members = [
 inherits = "release"
 strip = "symbols"
 lto = true
+
+[profile.profiling]
+inherits = "release"
+debug = "line-tables-only"
```

Comment:
This saves like 1GB of memory, compared to `debug = true`.

Reply:
seems reasonable. What about `debug = "limited"`?

Reply:
Didn't test with limited but when used with `parca` i didn't notice any difference between `true` and `"line-tables-only"`, so decided to select weakest possible option.

Reply:
makes sense